backend/pyproject.toml

```diff
@@ -48,6 +42,7 @@ include = ["zeno_backend"]
 
 [tool.ruff]
 select = ["E", "F", "I", "PD", "N", "D", "T20"]
+ignore = ["D206", "E501"]
```

Comment:
Why do we ignore these? Esp. line too long seems like something we want in code. Also should probably add comments on what these ignores do.

Reply:
Did some reading, this will still lint and format long lines, the different specs just have different definitions. Added comment.